package.json

```diff
@@ -74,6 +72,7 @@
     "react-transform-hmr": "1.0.1",
     "reconnectingwebsocket": "1.0.0",
     "run-sequence": "1.1.2",
+    "selenium-standalone": "6.15.3",
```

Comment:
Why have we added this?

Reply:
Because the "gulp-tasks" package requires this dependency directly in "node_modules". Previously it was enough to have the dependency in "gulp-tasks" but newer versions of npm create a hierarchy inside "node_modules" so this didn't work anymore.